metrics/opencensus.go

```diff
@@ -0,0 +1,40 @@
+package metrics
+
+import (
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// Keys
+var (
+	// TODO: figure out some appropriate keys
+	// KeyProtocol, _     = tag.NewKey("libp2p_protocol")
+	KeyMessageType, _ = tag.NewKey("message_type")
+)
+
+var (
+	MRpcReceivedMessages = stats.Int64("libp2p.io/dht/kad/rpc_received_messages", "Total number of messages received per RPC", stats.UnitDimensionless)
+	MRpcReceivedBytes    = stats.Int64("libp2p.io/dht/kad/rpc_received_bytes", "Total received bytes per RPC", stats.UnitBytes)
+	MRpcLatencyMs        = stats.Float64("libp2p.io/dht/kad/rpc_latency_ms", "Latency per RPC", stats.UnitMilliseconds)
+	RpcReceivedMessages  = &view.View{
```

Comment:
nitpick: instead of prefixing the measurements with `M`, appending the views with `View` would be my preference.

Reply:
I went with the tutorial recommendation here, for the most part the "measures" are the second class citizens for namespacing, consumers will only mostly be interested in the views.

Reply:
I think I prefer View too, let us add a few other useful metrics first and see if a pattern emerges.